### ISSUE TITLE
Features in README should be listed in detail

### ISSUE THREAD

**Issue by marcinjahn** [vikdevelop/SaveDesktop#105]
The [Readme](https://github.com/vikdevelop/SaveDesktop/blob/main/README.md) contains the listing of features:

```
SaveDesktop can save:

    your icons, fonts, and themes
    your settings
    your backgrounds (including dynamic wallpapers, provided that the same username is retained)
    your GNOME and Nautilus extensions
    your installed Flatpak apps

and more...
```

I think it should list exactly all the things that it allows to export/import. "and more" is too vague.

**Comment by vikdevelop** [comment]
Hello,
I updated README, what do you think about commit [`d64cc1c`](https://github.com/vikdevelop/SaveDesktop/commit/d64cc1c) ?

**Comment by marcinjahn** [comment]
I think it still is a bit too general, I guess that’s because some config files might contain stuff you might not know about? Then, maybe listing those files/directories would be a solution.

**Comment by vikdevelop** [comment]
I added a list of configuration files that will be included in the archive in [`e4b765d`](https://github.com/vikdevelop/SaveDesktop/commit/e4b765d).

**Comment by marcinjahn** [comment]
That looks awesome
